algeria/researcher_profile/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+
+class Institution(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Department(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Position(models.Model):
+    name = models.CharField(max_length=255)
+    
+    def __str__(self):
+        return self.name
+
+class Research_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Collaboration_Area(models.Model): ####implement fuzzy matching / standardization of names
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Link(models.Model):
+    link = models.URLField(max_length = 255, null=True, blank=True)
+    link_type = models.CharField(max_length=255)
+    link_type = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+
+class Profile(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.URLField(max_length=255, null=True, blank=True)
+    institution = models.ManyToManyField(Institution)
+    department = models.ManyToManyField(Department)
+    position = models.ManyToManyField(Position)
+    email = models.EmailField
+    resume_cv_link = models.URLField(max_length = 255, null=True, blank=True)
+    professional_links = models.ManyToManyField(Link)
```

Comment:
A `ManyToManyField` would make sense if the same link is used for multiple profiles, but b/c these are meant to be individual, personal links, I don't think this makes sense.  Instead of having a `professional_links` field, rather add a foreign key field in the `Links` table that points to `Profile` and name it `profile`.  Also, give it a `related_name="links"` so we can reverse-query if we have a Profile object.